lib/assertions.js

```diff
@@ -690,7 +690,7 @@ module.exports = function (expect) {
         '<object> to be (a map|a hash|an object) whose values [exhaustively] satisfy <assertion>'
     ], function (expect, subject, nextArg) {
         expect.errorMode = 'nested';
-        expect(subject, 'not to equal', {});
+        expect(subject, 'not to equal', {}); // TODO: to be non-empty
```

Comment:
This made me realize that `to have values satisfying` currently allows an empty array. Fixed here: https://github.com/unexpectedjs/unexpected/pull/349 (sorry for causing a conflict)